src/main/java/redis/embedded/util/OsArchitecture.java

```diff
@@ -6,7 +6,8 @@
     
     public static final OsArchitecture WINDOWS_x86 = new OsArchitecture(OS.WINDOWS, Architecture.x86);
     public static final OsArchitecture WINDOWS_x86_64 = new OsArchitecture(OS.WINDOWS, Architecture.x86_64);
-    
+    public static final OsArchitecture UNIX_aarch64 = new OsArchitecture(OS.UNIX, Architecture.aarch64);
+
```

Comment:
Check why they have added empty line and where it must be

Reply:
not getting why they have added the empty line as it does not have much effect on the code